tests/soak-tests/src/test/java/org/apache/activemq/artemis/tests/soak/brokerConnection/mirror/ReplicatedBothNodesMirrorTest.java

```diff
@@ -22,6 +22,7 @@
 import javax.jms.JMSException;
 import javax.jms.MessageConsumer;
 import javax.jms.MessageProducer;
+import javax.jms.Queue;
```

Comment:
General comment about the class, not this specific change.

This test class failed in the CI run you did, perhaps something to look at.

Reply:
I did another run and it failed again. This change seemingly introducing a failure, in a test method it altered the behaviour of, hardly seems orthogonal to this change.

Reply:
the files reporting on lsof are not part of this test. 

Looking at the CI output surefire is keeping files open apparently.


Running the test locally does not cause a failure. the test should only be accounted for the difference of lsof.. hence the change I said it needs to be done, but it's orthogonal to this PR.

Reply:
Some of them aren't from this test, but some clearly are for this test (look at the end, immediately after the truncation, for >100 lines for this specific test ). You cant see the majority because of the log truncation.

What can be said is that its failing by multiple thousands of files (expecting <4k, got almost 8k), there arent nearly that many test classes, it didnt fail before this change, but does after,  and the test [method] failing every time is one you are changing in a way that causes increased file usage. That failure isnt unrelated to this PR.

Reply:
@gemmellr  I will figure out what's going on... I'm hoping thee will clear out after I removed the retry parameters.
And BTW: the test now is clean from retries... I ran the test for 300 on my home server, plus another 200 on my laptop.. and no failures! So the flush I added  is working as expected.

Reply:
I reproduced the failures on my linux desktop.. I will fix it tomorrow

Reply:
lol the leaks that caused it was the Web interface. disabled and test will pass.